set.mm

```diff
@@ -632273,6 +632273,103 @@ fixed reference functional determined by this vector (corresponding to
+      QUVTYQWTGUXRYQXAYQXBUOVQXQUWTUWSUXRUXQVTUVTUXCUXBEYQXDVKWFUQXPXQUVTXRUXQU
+      XPXEXPXQUVTUDZXRUXQUXPUXSDVSGZUWTYRVSGZUDXRUXQIUXPXEUXSUXTUWTUYAXPXQUXTUV
+      TDWAXFXQXPUWTUVTUXCXGXQUVTUYAXPXQUWTUWSUYAUVTUXCUXBEYQXHVKXIXCDEYRXJUOXKX
+      LXMXCDYRUTWGVLXSXTUVTUVPUXKGXNUXEVCWJUKVDUPXO $.
+  $}
+
+  ${
+    $d N x $.  $d X x y $.  $d Y x y $.
+    $( A factorial with offset is monotonely increasing (Contributed by
+       metakunt, 20-Apr-2024.) $)
+    factwoffsmonot $p |- ( ( ( X e. NN0 /\ Y e. NN0 /\ X <_ Y ) /\ N e. NN0 )
+                      -> ( ! ` ( X + N ) ) <_ ( ! ` ( Y + N ) ) ) $=
+      ( cn0 wcel cle wbr caddc co cfa cfv cc0 c1 wceq oveq2 fveq2d syl wa jca
+      cr vx vy w3a cv breq12d facwordi cc nn0cn addid1 3ad2ant1 3ad2ant2 ax-1cn
+      3brtr4d addass mp3an3 syl2an 3ad2antl1 adantr nn0addcl 3adant2 3adant1 wb
+      cmul nn0re leadd1 syl3an biimpa syl3anc 3an1rs 1re mpbid wi cn faccl nnre
+      3syl clt nngt0 0re ltle mpan mpd 1nn0 nn0ge0 nn0readdcl readdcld lemul12a
+      mpan2 1red 3expa 3adantl3 facp1 breq12 3ad2antl2 breqtrd eqbrtrrd nn0indd
+      mpbird ) BDEZCDEZBCFGZUCZBUAUDZHIZJKZCXCHIZJKZFGBLHIZJKZCLHIZJKZFGBUBUDZH
+      IZJKZCXLHIZJKZFGZBXLMHIZHIZJKZCXRHIZJKZFGBAHIZJKZCAHIZJKZFGUAUBAXCLNZXEXI
+      XGXKFYGXDXHJXCLBHOPYGXFXJJXCLCHOPUEXCXLNZXEXNXGXPFYHXDXMJXCXLBHOPYHXFXOJX
+      CXLCHOPUEXCXRNZXEXTXGYBFYIXDXSJXCXRBHOPYIXFYAJXCXRCHOPUEXCANZXEYDXGYFFYJX
+      DYCJXCABHOPYJXFYEJXCACHOPUEXBBJKZCJKZXIXKFBCUFWSWTXIYKNXAWSXHBJWSBUGEZXHB
+      NBUHZBUIQPUJWTWSXKYLNXAWTXJCJWTCUGEZXJCNCUHZCUIQPUKUMXBXLDEZRZXQRZXMMHIZJ
+      KZXTYBFYRUUAXTNZXQWSWTYQUUBXAWSYQRZYTXSJWSYMXLUGEZYTXSNZYQYNXLUHZYMUUDMUG
+      EZUUEULBXLMUNUOUPPUQURYSUUAXOMHIZJKZYBFYRUUAUUIFGZXQYRUUJXNYTVCIZXPUUHVCI
+      ZFGZYRXQYTUUHFGZRZUUMYRXQUUNWSWTYQXAXQWSWTYQUCZXARZXMDEZXODEZXMXOFGZXQUUP
+      UURXAWSYQUURWTBXLUSZUTURZUUPUUSXAWTYQUUSWSCXLUSZVAURZUUPXAUUTWSBTEWTCTEYQ
+      XLTEXAUUTVBBVDCVDXLVDBCXLVEVFVGZXMXOUFVHVIWSWTYQXAUUNUUQUUTUUNUVEUUQXMTEZ
+      XOTEZRUUTUUNVBZUUQUVFUVGUUQUURUVFUVBXMVDQUUQUUSUVGUVDXOVDQSUVFUVGMTEUVHVJ
+      XMXOMVEUOQVKVISWSWTYQUUOUUMVLZXAWSWTYQUVIUUPXNTEZLXNFGZRZXPTEZRZYTTEZLYTF
+      GZRZUUHTEZRZRUVIUUPUVNUVSUUPUVLUVMUUPUVJUVKWSYQUVJWTUUCUURXNVMEZUVJUVAXMV
+      NZXNVOVPZUTWSYQUVKWTUUCLXNVQGZUVKUUCUURUVTUWCUVAUWAXNVRVPUUCUVJUWCUVKVLZU
+      WBLTEUVJUWDVSLXNVTWAQWBUTSWTYQUVMWSWTYQRZUUSXPVMEUVMUVCXOVNXPVOVPVASUUPUV
+      QUVRUUPUVOUVPWSYQUVOWTUUCYTDEZUVOUUCUURUWFUVAUURMDEUWFWCXMMUSWHQZYTVDQUTW
+      SYQUVPWTUUCUWFUVPUWGYTWDQUTSWTYQUVRWSUWEXOMCXLWEUWEWIWFVASSXNXPYTUUHWGQWJ
+      WKWBWSWTYQXAUUJUUMVBZUUQUUAUUKNZUUIUULNZRUWHUUQUWIUWJUUQUURUWIUVBXMWLQUUQ
+      UUSUWJUVDXOWLQSUUAUUKUUIUULFWMQVIWRURYRUUIYBNZXQWTWSYQUWKXAUWEUUHYAJWTYOU
+      UDUUHYANZYQYPUUFYOUUDUUGUWLULCXLMUNUOUPPWNURWOWPWQ $.
+  $}
+
+  ${
+    $( 2x+3 is greater than x+2 for natural numbers (Contributed by metakunt,
+       20-Apr-2024.) $)
+    2xp3dxp1ge1 $p |- ( x e. NN -> 1 <_ ( ( ( 2 x. x ) + 3 ) / ( x + 2 ) ) ) $=
```

Comment:
` 1 <_ ( ( ( 2 x. x ) + 3 ) / ( x + 2 ) ) ) ` means `2x+3` is greater than **or equal to** ` x+2 `. So either the comment has to be corrected, or the theorem itsself. In the first case, the theorem would hold for `( -u 1 [,) +oo )` (see also @tirix `s comment).

Reply:
I am working on correcting the theorem, however I get a very weird message when I try to import it into metamath from mmj2. Here is the mmj2 file
```
h50::2xp3dxp2ge1d.1   |- ( ph -> X e. ( -u 1 [,) +oo ) )


51::2cn |- 2 e. CC
52::ax-1cn |- 1 e. CC
*32:30,31:pm3.2i |- ( 2 e. CC /\ 1 e. CC  )
53::neg1cn |- -u 1 e. CC
54::0re |- 0 e. RR
55::1re |- 1 e. RR 
56::0lt1 |-  0 < 1 
57::3re |- 3 e. RR
58::recn |- ( X e. RR -> X e. CC )
59::elicopnf       |- (  -u 1 e. RR
                      -> (   X e. ( -u 1 [,) +oo )
                         <-> ( X e. RR /\ -u 1 <_ X ) ) )
60::neg1rr |- -u 1 e. RR
61:60,59:ax-mp     |- (   X e. ( -u 1 [,) +oo )  <-> ( X e. RR /\ -u 1 <_ X ) )
62:50,61:sylib |- ( ph -> ( X e. RR /\ -u 1 <_ X ) )
63:62:simpld |-  ( ph ->  X e. RR )
64:62:simprd |-  ( ph ->   -u 1 <_ X  )
65::2re |- 2 e. RR 
66::readdcl |- ( ( X e. RR /\ 2 e. RR ) -> ( X + 2 ) e. RR )
67:65,66:mpan2 |- (  X e. RR -> ( X + 2 ) e. RR )
68:63,67:syl       |- ( ph -> ( X + 2 ) e. RR )
69::leadd1         |- (  ( -u 1 e. RR /\ X e. RR /\ 2 e. RR ) -> (   -u 1 <_ X <-> ( -u 1 + 2 ) <_ ( X + 2 ) ) )
70:60,65,69:mp3an13  |- (  X e. RR -> (   -u 1 <_ X <-> ( -u 1 + 2 ) <_ ( X + 2 ) ) )
71:63,70:syl       |- (  ph -> (   -u 1 <_ X  <-> ( -u 1 + 2 ) <_ ( X + 2 ) ) )
72:64,71:mpbid     |- (  ph  -> ( -u 1 + 2 ) <_ ( X + 2 ) )
73::addcom         |- (  ( -u 1 e. CC /\ 2 e. CC ) -> ( -u 1 + 2 ) = ( 2 + -u 1 ) )
74:53,51,73:mp2an |-  ( -u 1 + 2 ) = ( 2 + -u 1 ) 
75::negsub         |- (  ( 2 e. CC /\ 1 e. CC ) -> ( 2 + -u 1 ) = ( 2 - 1 ) )
76:51,52,75:mp2an |- ( 2 + -u 1 ) = ( 2 - 1 )
77::2m1e1 |- ( 2 - 1 ) = 1
78:74,76,77:3eqtri |- ( -u 1 + 2 ) = 1
79:78,72:eqbrtrrid |- (  ph  ->  1 <_ ( X + 2 ) ) 


80::ltletr         |- (  ( 0 e. RR /\ 1 e. RR /\ ( X + 2 ) e. RR )
                      -> (  ( 0 < 1 /\ 1 <_ ( X + 2 ) )
                         -> 0 < ( X + 2 ) ) )
81::ltletr |- (  ( 0 e. RR /\ 1 e. RR /\ ( X + 2 ) e. RR )
                      -> (  ( 0 < 1 /\ 1 <_ ( X + 2 ) )
                         -> 0 < ( X + 2 ) ) )
82:54,55,81:mp3an12 |-  (   ( X + 2 ) e. RR -> (  ( 0 < 1 /\ 1 <_ ( X + 2 ) ) -> 0 < ( X + 2 ) ) )
83:68,82:syl       |- (  ph -> (  ( 0 < 1 /\ 1 <_ ( X + 2 ) ) -> 0 < ( X + 2 ) ) )
84:79,56:jctil |- (  ph ->  ( 0 < 1 /\ 1 <_ ( X + 2 ) ) )
85:84,83:mpd |- (  ph -> 0 < ( X + 2 ) ) 
86::elrp           |- (   ( X + 2 ) e. RR+ <-> ( ( X + 2 ) e. RR /\ 0 < ( X + 2 ) ) )
87:86:imbi2i |-  (  ( ph -> ( X + 2 ) e. RR+ ) <-> ( ph -> ( ( X + 2 ) e. RR /\ 0 < ( X + 2 ) ) ) )
88:68,85:jca |- ( ph -> ( ( X + 2 ) e. RR /\ 0 < ( X + 2 ) ) )
89:63,58:syl |- ( ph -> X e. CC )

90::remulcl         |- ( ( 2 e. RR /\ X e. RR ) -> ( 2 x. X ) e. RR )
91:65,90:mpan   |-  (   X e. RR  -> ( 2 x. X ) e. RR )
92:63,91:syl       |- ( ph -> ( 2 x. X ) e. RR )
93::readdcl        |- (  ( ( 2 x. X ) e. RR /\ 3 e. RR ) -> ( ( 2 x. X ) + 3 ) e. RR )
94:57,93:mpan2   |- (  (  2 x. X ) e. RR -> ( ( 2 x. X ) + 3 ) e. RR )

95::leadd1         |- (  ( -u 1 e. RR /\ X e. RR /\ 1 e. RR )
                      -> (   -u 1 <_ X <-> ( -u 1 + 1 ) <_ ( X + 1 ) ) )
96:60,55,95:mp3an13   |- (   X e. RR  -> (   -u 1 <_ X <-> ( -u 1 + 1 ) <_ ( X + 1 ) ) )
97:63,96:syl       |- (  ph -> (   -u 1 <_ X  <-> ( -u 1 + 1 ) <_ ( X + 1 ) ) )
98:64,97:mpbid   |- (  ph -> ( -u 1 + 1 ) <_ ( X + 1 ) ) 
99:52:negidi       |- ( 1 + -u 1 ) = 0
100:53,52:addcomi  |- ( -u 1 + 1 ) = ( 1 + -u 1 )
101:100,99:eqtri  |- ( -u 1 + 1 ) = 0
102:101,98:eqbrtrrid  |- (  ph -> 0 <_ ( X + 1 ) ) 
103::readdcl       |- ( ( X e. RR /\ 1 e. RR ) -> ( X + 1 ) e. RR )
104:55,103:mpan2 |- (  X e. RR -> ( X + 1 ) e. RR )
105:63,104:syl |- ( ph -> ( X + 1 ) e. RR )
106::leadd2        |- (  ( 0 e. RR /\ ( X + 1 ) e. RR /\ X e. RR ) -> (   0 <_ ( X + 1 ) <-> ( X + 0 ) <_ ( X + ( X + 1 ) ) ) )
107:54,106:mp3an1 |- (  ( ( X + 1 ) e. RR /\ X e. RR ) -> (   0 <_ ( X + 1 ) <-> ( X + 0 ) <_ ( X + ( X + 1 ) ) ) )
108:105,63:jca     |- (  ph -> ( ( X + 1 ) e. RR /\ X e. RR ) )
109:108,107:syl    |- (  ph -> (   0 <_ ( X + 1 ) <-> ( X + 0 ) <_ ( X + ( X + 1 ) ) ) )
110:102,109:mpbid |- (  ph -> ( X + 0 ) <_ ( X + ( X + 1 ) ) ) 
111::addid1        |- ( X e. CC -> ( X + 0 ) = X )
112:58,111:syl   |- ( X e. RR -> ( X + 0 ) = X )
113:63,112:syl     |- ( ph -> ( X + 0 ) = X )
114::addass        |- (  ( X e. CC /\ X e. CC /\ 1 e. CC ) -> ( ( X + X ) + 1 ) = ( X + ( X + 1 ) ) )
115:52,114:mp3an3 |- (  ( X e. CC /\ X e. CC ) -> ( ( X + X ) + 1 ) = ( X + ( X + 1 ) ) )
116:115:anidms |- (   X e. CC -> ( ( X + X ) + 1 ) = ( X + ( X + 1 ) ) )
117:89:2timesd     |- ( ph -> ( 2 x. X ) = ( X + X ) )
118:89,116:syl     |- (  ph -> ( ( X + X ) + 1 ) = ( X + ( X + 1 ) ) )
119:117:oveq1d     |- (  ph -> ( ( 2 x. X ) + 1 ) = ( ( X + X ) + 1 ) )
120:119,118:eqtrd |- (  ph -> ( ( 2 x. X ) + 1 ) = ( X + ( X + 1 ) ) )
121:110,120:breqtrrd |- (  ph -> ( X + 0 ) <_ ( ( 2 x. X ) + 1 ) ) 
122:113,121:eqbrtrrd |- (  ph ->  X  <_ ( ( 2 x. X ) + 1 ) ) 

123:65:a1i         |- ( ph -> 2 e. RR )
124::1red          |- ( ph -> 1 e. RR )
125:92,124:readdcld |- (  ph
                      -> ( ( 2 x. X ) + 1 ) e. RR )
126:123:leidd      |- ( ph -> 2 <_ 2 )
127:63,123,125,123,122,126:le2addd 
                   |- (  ph -> ( X + 2 ) <_ ( ( ( 2 x. X ) + 1 ) + 2 ) )

128:92:recnd       |- ( ph -> ( 2 x. X ) e. CC )
129:52:a1i         |- ( ph -> 1 e. CC )
130:51:a1i         |- ( ph -> 2 e. CC )
131:128,129,130:addassd 
                   |- (  ph ->   ( ( ( 2 x. X ) + 1 ) + 2 )  = ( ( 2 x. X ) + ( 1 + 2 ) ) )
132::1p2e3 |- ( 1 + 2 ) = 3
133::oveq2         |- (  ( 1 + 2 ) = 3 -> ( ( 2 x. X ) + ( 1 + 2 ) ) = ( ( 2 x. X ) + 3 ) )
134:132,133:ax-mp  |- ( ( 2 x. X ) + ( 1 + 2 ) ) = ( ( 2 x. X ) + 3 )
135:131,134:syl6eq |- (  ph ->   ( ( ( 2 x. X ) + 1 ) + 2 )  = ( ( 2 x. X ) + 3 ) )
136:127,135:breqtrd |- (  ph -> ( X + 2 ) <_ ( ( 2 x. X ) + 3 ) )
*84:: |-  (  ph -> (  ( 0 < 1 /\ 1 <_ ( X + 2 ) ) -> 0 < ( X + 2 ) ) )
137:88,87:mpbir |- ( ph -> ( X + 2 ) e. RR+ )

138:92,94:syl   |- (  ph   -> ( ( 2 x. X ) + 3 ) e. RR )

139:137,138,136:3jca |- ( ph -> (  ( X + 2 ) e. RR+ /\ ( ( 2 x. X ) + 3 ) e. RR /\ ( X + 2 ) <_ ( ( 2 x. X ) + 3 ) ) )
140::divge1        |- (  (  ( X + 2 ) e. RR+ /\ ( ( 2 x. X ) + 3 ) e. RR /\ ( X + 2 ) <_ ( ( 2 x. X ) + 3 ) )
                      -> 1 <_ ( ( ( 2 x. X ) + 3 ) / ( X + 2 ) ) )
qed:139,140:syl           |- ( ph -> 1 <_ ( ( ( 2 x. X ) + 3 ) / ( X + 2 ) ) ) 


$=  ( c2 caddc co wcel c3 cr cle wbr c1 wi cc0 wa 2re syl wceq ax-1cn
    wb cc 2xp3dxp2ge1d.1 crp cmul cdiv cneg cpnf cico neg1rr elicopnf
    w3a clt ax-mp sylib simpld readdcl mpan2 cmin neg1cn addcom mp2an
    2cn negsub 2m1e1 3eqtri simprd mp3an13 mpbid eqbrtrrid 0lt1 jctil
    leadd1 0re 1re ltletr mp3an12 mpd jca elrp mpbir remulcl mpan 3re
    imbi2i a1i 1red readdcld recn addid1 addcomi negidi leadd2 mp3an1
    eqtri 2timesd oveq1d addass mp3an3 anidms eqtrd breqtrrd eqbrtrrd
    leidd le2addd recnd addassd 1p2e3 oveq2 syl6eq breqtrd divge1
    3jca ) ABCDEZUBFZCBUCEZGDEZHFZXLXOIJZUJKXOXLUDEIJAXMXPXQAXMLAXLHF
    ZMXLUKJZNZLAXRXSABHFZXRAYAKUEZBIJZABYBUFUGEFZYAYCNZUAYBHFZYDYESUH
    YBBUIULUMZUNZYACHFZXROBCUOUPPZAMKUKJZKXLIJZNZXSAYLYKAKYBCDEZXLIYN
    CYBDEZCKUQEZKYBTFCTFZYNYOQURVAYBCUSUTYQKTFZYOYPQVARCKVBUTVCVDAYCY
    NXLIJZAYAYCYGVEZAYAYCYSSZYHYFYAYIUUAUHOYBBCVKVFPVGVHVIVJAXRYMXSLZ
    YJMHFZKHFZXRUUBVLVMMKXLVNVOPVPVQXMXTAXLVRWCVSAXNHFZXPAYAUUEYHYIYA
    UUEOCBVTWAPZUUEGHFXPWBXNGUOUPPAXLXNKDEZCDEZXOIABCUUGCYHYIAOWDZAXN
    KUUFAWEWFUUIABMDEZBUUGIAYAUUJBQZYHYABTFZUUKBWGZBWHPPAUUJBBKDEZDEZ
    UUGIAMUUNIJZUUJUUOIJZAMYBKDEZUUNIUURKYBDEMYBKURRWIKRWJWMAYCUURUUN
    IJZYTAYAYCUUSSZYHYFYAUUDUUTUHVMYBBKVKVFPVGVHAUUNHFZYANUUPUUQSZAUV
    AYAAYAUVAYHYAUUDUVAVMBKUOUPPYHVQUUCUVAYAUVBVLMUUNBWKWLPVGAUUGBBDE
    ZKDEZUUOAXNUVCKDABAYAUULYHUUMPZWNWOAUULUVDUUOQZUVEUULUVFUULUULYRU
    VFRBBKWPWQWRPWSWTXAACUUIXBXCAUUHXNKCDEZDEZXOAXNKCAXNUUFXDYRARWDYQ
    AVAWDXEUVGGQUVHXOQXFUVGGXNDXGULXHXIXKXLXOXJP $.
$)
```

As metamath says, I have removed the explicit hypotheses manually and this is what I paste into set.mm
```
  ${
    2xp3dxp2ge1d.1 $e |- ( ph -> X e. ( -u 1 [,) +oo ) ) $.
    $( 2x+3 is greater than x+2 for x >= -1, a deduction version (Contributed
       by metakunt, 21-Apr-2024.) $)
    2xp3dxp2ge1d $p |- ( ph -> 1 <_ ( ( ( 2 x. X ) + 3 ) / ( X + 2 ) ) ) $=
    ( c2 caddc co wcel c3 cr cle wbr c1 wi cc0 wa 2re syl wceq ax-1cn
    wb cc crp cmul cdiv cneg cpnf cico neg1rr elicopnf
    w3a clt ax-mp sylib simpld readdcl mpan2 cmin neg1cn addcom mp2an
    2cn negsub 2m1e1 3eqtri simprd mp3an13 mpbid eqbrtrrid 0lt1 jctil
    leadd1 0re 1re ltletr mp3an12 mpd jca elrp mpbir remulcl mpan 3re
    imbi2i a1i 1red readdcld recn addid1 addcomi negidi leadd2 mp3an1
    eqtri 2timesd oveq1d addass mp3an3 anidms eqtrd breqtrrd eqbrtrrd
    leidd le2addd recnd addassd 1p2e3 oveq2 syl6eq breqtrd divge1
    3jca ) ABCDEZUBFZCBUCEZGDEZHFZXLXOIJZUJKXOXLUDEIJAXMXPXQAXMLAXLHF
    ZMXLUKJZNZLAXRXSABHFZXRAYAKUEZBIJZABYBUFUGEFZYAYCNZUAYBHFZYDYESUH
    YBBUIULUMZUNZYACHFZXROBCUOUPPZAMKUKJZKXLIJZNZXSAYLYKAKYBCDEZXLIYN
    CYBDEZCKUQEZKYBTFCTFZYNYOQURVAYBCUSUTYQKTFZYOYPQVARCKVBUTVCVDAYCY
    NXLIJZAYAYCYGVEZAYAYCYSSZYHYFYAYIUUAUHOYBBCVKVFPVGVHVIVJAXRYMXSLZ
    YJMHFZKHFZXRUUBVLVMMKXLVNVOPVPVQXMXTAXLVRWCVSAXNHFZXPAYAUUEYHYIYA
    UUEOCBVTWAPZUUEGHFXPWBXNGUOUPPAXLXNKDEZCDEZXOIABCUUGCYHYIAOWDZAXN
    KUUFAWEWFUUIABMDEZBUUGIAYAUUJBQZYHYABTFZUUKBWGZBWHPPAUUJBBKDEZDEZ
    UUGIAMUUNIJZUUJUUOIJZAMYBKDEZUUNIUURKYBDEMYBKURRWIKRWJWMAYCUURUUN
    IJZYTAYAYCUUSSZYHYFYAUUDUUTUHVMYBBKVKVFPVGVHAUUNHFZYANUUPUUQSZAUV
    AYAAYAUVAYHYAUUDUVAVMBKUOUPPYHVQUUCUVAYAUVBVLMUUNBWKWLPVGAUUGBBDE
    ZKDEZUUOAXNUVCKDABAYAUULYHUUMPZWNWOAUULUVDUUOQZUVEUULUVFUULUULYRU
    VFRBBKWPWQWRPWSWTXAACUUIXBXCAUUHXNKCDEZDEZXOAXNKCAXNUUFXDYRARWDYQ
    AVAWDXEUVGGQUVHXOQXFUVGGXNDXGULXHXIXKXLXOXJP $.
  $}
```

But I get an error in the comment below, since the stack is not empty.